src/etl/bgi_etl.py

```diff
@@ -177,333 +183,443 @@ def _load_and_process_data(self):
+            genetic_entity_external_url = ""
+            mod_cross_reference_complete_url = ""
+            taxon_id = basic_genetic_entity.get("taxonId")
+            short_species_abbreviation = ETLHelper.get_short_species_abbreviation(taxon_id)
 
 
-            if basicGeneticEntity.get('taxonId') == "NCBITaxon:9606" or basicGeneticEntity.get('taxonId') == "NCBITaxon:10090":
-                local_id = basicGeneticEntity.get('primaryId')
+            if basic_genetic_entity.get('taxonId') in ["NCBITaxon:9606", "NCBITaxon:10090"]:
+                local_id = basic_genetic_entity.get('primaryId')
 
-            if self.testObject.using_test_data() is True:
-                is_it_test_entry = self.testObject.check_for_test_id_entry(primary_id)
+            if self.test_object.using_test_data() is True:
+                is_it_test_entry = self.test_object.check_for_test_id_entry(primary_id)
                 if is_it_test_entry is False:
                     counter = counter - 1
                     continue
 
-            if 'crossReferences' in basicGeneticEntity:
-                for crossRef in basicGeneticEntity.get('crossReferences'):
-                    if ':' in crossRef.get('id'):
-                        crossRefId = crossRef.get('id')
-                        localCrossRefId = crossRefId.split(":")[1]
-                        prefix = crossRef.get('id').split(":")[0]
-                        pages = crossRef.get('pages')
-                        globalXrefId = crossRef.get('id')
-                        displayName = globalXrefId
+            if 'crossReferences' in basic_genetic_entity:
+                for cross_ref in basic_genetic_entity.get('crossReferences'):
+                    if ':' in cross_ref.get('id'):
+                        cross_ref_id = cross_ref.get('id')
+                        local_cross_ref_id = cross_ref_id.split(":")[1]
+                        prefix = cross_ref.get('id').split(":")[0]
+                        pages = cross_ref.get('pages')
+                        global_xref_id = cross_ref.get('id')
+                        display_name = global_xref_id
 
                         # some pages collection have 0 elements
                         if pages is not None and len(pages) > 0:
                             for page in pages:
-                                modCrossReferenceCompleteUrl = ""
-                                geneLiteratureUrl = ""
-                                displayName = ""
+                                mod_cross_reference_complete_url = ""
+                                gene_literature_url = ""
+                                display_name = ""
 
-                                crossRefCompleteUrl = ETLHelper.get_page_complete_url(localCrossRefId, ETL.xrefUrlMap,
-                                                                                      prefix, page)
+                                cross_ref_complete_url = ETLHelper.get_page_complete_url(\
```

Comment:
can the first item in the map be in the same line as the function call?  I find this more readable than \